controller/src/beerocks/master/db/db.cpp

```diff
@@ -200,6 +200,25 @@ std::string db::get_node_ipv4(std::string mac)
     return n->ipv4;
 }
 
+bool db::set_node_manufacturer(std::string mac, std::string manufacturer)
```

Comment:
I know the rest of this file doen't follow this - but better to have the parameters as const. This applies to all newly added functions below.

Reply:
You made this comment in another review as well, but I don't understand it. Do you really want a signature like

```
void foo(const int x1, const bool yesno, const std::string bar);
```

?

Or do you only want const-ness for `std::string` and not for scalar types?

Or do you actually want a const ref `const std::string&` because you don't trust the copyref optimisation?

Reply:
I prefer a signature with const values only for non-scalar types whenever possible. Also the function should be declared `const` if it a class function and doesn't change any members:

```
void foo(,int x1, bool yesno, const std::string &str) const; //function const for class members when possible
```
If the parameter is inout, then it should be passed without const.
If the function returns scalar type, it should return none const (anyway copied).
If the function returns non-scalar type - then it should return const when possible to save an extra copy constructor call.

Of course all these are really (nitpicks), so don't bother changing anything if you disagree.
Also 
```